packages/subiquity_client/test/subiquity_client_test.dart

```diff
@@ -427,9 +427,6 @@ void main() {
       final stepPressKey = step0 as StepPressKey;
       expect(stepPressKey.symbols, isNotEmpty);
       expect(stepPressKey.keycodes, isNotEmpty);
-      expect(stepPressKey.keycodes!.first, hasLength(2));
-      expect(stepPressKey.keycodes!.first.first, isA<int>());
-      expect(stepPressKey.keycodes!.first.last, isA<String>());
```

Comment:
`stepPressKey.keycodes` was previously `List<List<dynamic>>` and this code tested that it had int and String elements. Now, `stepPressKey` is `Map<int, String>` so testing the types is no longer necessary.